store/index.js

```diff
@@ -20,24 +25,26 @@ export const mutations = {
   },
   addScore(state, { score }) {
     state.scores.push(score);
+  },
+  addLastScore(state, { score }) {
+    state.lastScore = score;
   }
 };
 
 export const actions = {
   async initScore({ commit }) {
     let newItem = false;
-    await refScores
-      // .endAt()
-      // .limitToLast(1)
-      .on('child_added', snapshot => {
-        if (!newItem) return;
-        commit('addScore', {
-          score: {
-            uid: snapshot.key,
-            ...snapshot.val()
-          }
-        });
-      });
+    await refScores.on('child_added', snapshot => {
```

Comment:
__[JSHint]__ <sub>posted by Sider</sub>

Expected an assignment or function call and instead saw an expression.